src/module.ts

```diff
@@ -18,8 +18,16 @@ export function resolveNitroModule(
       // @ts-ignore
       globalThis.defineNitroModule || defineNitroModule;
 
-    const _jiti = jiti(nitroOptions.rootDir, { interopDefault: true });
-    const _modPath = _jiti.resolve(resolvePath(mod, nitroOptions));
```

Comment:
`jiti.resolve` itself should be able to do the resolution (it uses mlly when in esm mode even!).

We might need to extend options part to pass `modulesDir` (which also contains `rootDir`)

Reply:
The issue may come from `resolvePath` which don't use mlly nor jiti to resolve the path. So `jiti.resolve` is actually receiving an absolute path to resolve.
Maybe removing it and let `jiti.resolve` do all the work of `resolvePath` would also work 🤔

Reply:
oh it does work by removing the `resolvePath` !

Reply:
Yes, we also need to support aliases btw `~/modules/`, etc. Wondering why these two are incompatible..

Reply:
there's an alias option in JITI. We could probably use it

Reply:
was it finally a bug in pathe? If too hard using jiti aliases should be also good idea but if easy solvable would be also nice to fix `resolvePath` because it is used for other keys and I worry behavior can be different in long term.

Reply:
Yes it was :+1:
I just tested this with a local link to the pr from pathe with this PR and everything seems to be solved
nitro -> link:../jiti -> link:../pathe